ui/components/MesheryResults.js

```diff
@@ -328,7 +328,8 @@ class MesheryResults extends Component {
           });
         } else {
           Object.keys(results_selection[page]).forEach((ind) => {
-            rowsSelected.push(ind);
+            const val = parseInt(ind)
```

Comment:
reason 🤔 ?

Reply:
Mui-datatables expects an array of integers and we were passing an array of strings.